scripts/ebeam_scan_to_multiprobe.py

```diff
@@ -105,10 +162,20 @@ def main(args):
         raise ValueError("Backend is not running.")
 
     ccd = model.getComponent(role=options.role)
-
+    scanner = model.getComponent(role="ebeam-control")
     try:
-        sem_rotation = get_sem_rotation(ccd)
-        print("Apply {:.3f}° to the scan rotation using SEM PC".format(math.degrees(sem_rotation)))
+        sem_rotation = get_sem_rotation(ccd, auto=options.auto,
+                                        channel=options.channel)
+        if options.auto:
+            val = scanner.rotation.value
+            scanner.rotation.value = val + sem_rotation
+            print("Added {:.3f}° to the scan rotation using SEM PC."
+                  "Rotation is now set to: {:.3f}°".format(
+                math.degrees(sem_rotation),
+                math.degrees(scanner.rotation.value)))
+        else:
+            print("Add {:.3f}° to the scan rotation using SEM PC".format(
+                math.degrees(sem_rotation)))
```

Comment:
How is adding the angle to the scan rotation done at the SEM PC. Is there a specific name in the control software. If yes maybe nice to add this info. 
Btw. Is the angle calculation including if the correction needs to be CW or CCW. I guess it does but didn't check explicitly.

Reply:
Nice! Btw what is the accuracy needed for the output?

Reply:
you can enter the rotation in the SEM software with 1 value after the point